modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+You can configure cloud credentials for the {cert-manager-operator} on the GCP Workload Identity cluster by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) of the {cert-manager-operator} by running the following command:
```

Comment:
Similar to https://github.com/openshift/openshift-docs/pull/59717#discussion_r1208926122
```suggestion
. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
```